charts/thoras/templates/api-server/webhooks.yaml

```diff
@@ -28,7 +28,7 @@ data:
 apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: "pods.thoras.ai"
+  name: "thoras.ai"
```

Comment:
@Rickyxstar assuming that changing the webhook name will cause us to create a new webhook instead of modifying the old, let's huddle to figure out how to ensure users who do a simple upgrade end up in a clean state with only one webhook

Reply:
I reverted the name change!